pipeline/setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup
```

Comment:
I'm following this: https://gist.github.com/ericmjl/27e50331f24db3e8f957d1fe7bbbe510. The only exceptions are the main.py and that I placed tests in a separate folder to keep them organized. I'll comment on the main.py below.

There's also this: https://packaging.python.org/en/latest/tutorials/packaging-projects/. However, I think some parts of the tutorial are not relevant to the risteys pipeline, since the python package only meant to be used by us.

Reply:
Thanks for the resources!

I am not 100% sure, but from my understanding: since we don't need to publish the package (and actually can't get it from within the sandboxed environments we use to run pipeline), then we can simply use one (ore more) `__init__.py` to make the import work for the pipeline scripts, and we don't need the `setup.py`.
https://docs.python.org/3/tutorial/modules.html#packages

Or do you see any reason to use the setup.py for our use cases?

Reply:
I think either way is good. Here's a nice (brief) description of the benefits of `setup.py` that I found today: https://goodresearch.dev/setup.html#install-a-project-package. I think for this project the main benefit would be that using the functions within notebooks (for testing/experimentation) is a bit easier.